tab-oneipl/minutes/2022_02_17_Minutes.rst

```diff
@@ -0,0 +1,186 @@
+   * If you have device allocation and memory assess on device -> no copy is
+     performed.
+   * If you have shared allocator and access for read -> no copy as well, but
+     for the write access copy to host would happen after kernel is finished.
+   * In image_allocator_t case the memory will be allocated on both host
+     and device and access is possible only to write or to read, but not both.
+     In case of read no copy to host happen, in case of write copy to host
+     would happen after kernel is finished.
+
+* **Ashish Uthama, Mathworks**: Actually is this SYCL limitation? It is not so
+  clear: if you have reading only, you do not know if it do the copy or not.
+  I prefer the explicit copy operation, so I am wondering if this is
+  restriction of SYCL?
+
+  **Victor Getmanskiy, Intel**: For the USM it is possible to implement
+  explicit copy, for Image it is not controlled (SYCL limitation). You still
+  can explicitly copy USM memory and create image with device_allocator_t over
+  memory without copy.
+
+  For Images (Textures) creating an object over device memory is currently not
+  possible because of SYCL limitations, the construction from host pointer is
+  only available. But we are working on it as well.
+
+* **Ashish Uthama, Mathworks**: I have a question about device context and
+  different data allocators usage. For example, If I have integration to a 3rd
+  party library, and some data is allocated, is it possible to get the same
+  context?
+
+**Victor Getmanskiy, Intel**: This is not done in the library, but there is
+  technical possibility to obtain the context, for example CUDA backends in
+  oneMKL open-source are capable to work over context of different devices like
+  NVIDIA. I think, in this case memory type will be device and mapping over
+  such memory with proper queue would create image over memory without extra
+  copy operation.
+
+* In spec ver 0.5 default allocator was implementation-defined.
+  For compatibility, in spec ver 0.6 the default allocator is selected as
+  shared USM.
+
+
+oneIPL Domains:
+
+* On the slide 15, the domains in blue are currently described in the oneIPL
+  Specification, yellow color is to be considered in the future versions of
+  spec.
+
+* Color conversion domain includes conversion functions for the main formats,
+  the YUV422 and YUV444 to rgb are considered as a possible future extension.
+
+* The Color conversions Domain API's are presented.
+
+* The Filtering Domain in oneIPL spec ver 0.6 includes Sobel filter with the
+  fixed 3x3 kernel size and Gaussian filter. Possible future extension of
+  oneIPL spec is considered to include Bilateral, Sobel, Filter Box and Filter
+  Median.
+
+* The Transformation Domain includes resize functions with bilinear, bicubic,
+  lanczos and supersampling interpolation types and also mirroring. Future
+  extensions are: the Nearest Resize and Warp Affine bilinear functions.
```

Comment:
it is better to rephrase as "nearest neighbor resize"

Reply:
fixed, thanks